extensions/facets/standard/clause_6_overview.adoc

```diff
@@ -3,38 +3,40 @@
 
 A **facet** refers to a high-level piece of information that is computed over a set of records in a given collection.  A facet differs from a search; it is used as a means to provide a better understanding of the composition of search results, and aids the users to refine and narrow their search.
 
+A useful function of facets is to allow users to "drill-down" on the values they report, thus offering an easy way to refine a search request and narrow down results. This is done by advertising to the client which property was used to compute the facet; by combining the property and desired value, a new search filter can be produced by the client.
+
 A facet can be represented by several mechanisms, as described below.
 
 === Term facet
 
-A term facet can be applied to any text property and produces a list of values appearing for a specific property across all matching records, as well as the count of records containing each value.
+A term facet can be applied to any text property and produces a list of unique values appearing for a specific property across all matching records, as well as the count of records containing each value.
 
 ==== Example
 
-A common use case for a term facet involves identifying results by organizations.  A term facet applies against contact organizations would provides term facet list per below:
+A common use case for a term facet involves identifying results by organizations.  A term facet applied against a `properties.contacts[].organization` property would provide a term facet list per below:
 
-* `Meteorological Service of Canada` (52 results)
-* `National Oceanic and Atmospheric Administration` (45 results)
-* `UK Met Office` (33 results)
+* "Meteorological Service of Canada" (52 results)
```

Comment:
while we are here, can we `s/results/records/` (to be consistent with the examples further in the section)?